lte/gateway/c/core/oai/tasks/nas5g/src/M5GULNASTransport.cpp

```diff
@@ -139,7 +138,7 @@ int ULNASTransportMsg::DecodeULNASTransportMsg(
         break;
     }
     if (decoded_result < 0) {
-      MLOG(MDEBUG) << "ERROR: decode ULNASTransportMsg failed";
+      OAILOG_ERROR(LOG_NAS5G, "ULNASTransportMsg Decoding FAILED");
```

Comment:
This file should also include the oai log header directly, if it's not already doing so.